lib/omniauth/strategies/openid_connect.rb

```diff
@@ -44,6 +44,7 @@ class OpenIDConnect
       option :send_scope_to_token_endpoint, true
       option :client_auth_method
       option :post_logout_redirect_uri
+      option :extra_authorize_params, {}
```

Comment:
I would go for `:optional_params` instead, but I won't fight for it :slightly_smiling_face:

Reply:
I chose this name because of the original repo.
I want to keep it as close to the original repo as possible.

Reply:
Fine by me.